### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by mohametalmeari**
- [x] Create app.rb file that serves as console app entry-point
- [x] In main.rb define the entry point

### CODE REVIEW

**Comment by mohametalmeari** [comment]
Hi @Gambit142, thank you for the review. However, I couldn't fix the last linter error, which is why I requested a review. Could you please guide me on how to make the main.rb file executable?

File: main.rb [hunk 1]
```diff
@@ -1,67 +1,48 @@
...
-require './trimmer_decorator'
-require './capitalize_decorator'
-require './person'
-require './teacher'
-require './student'
-require './classroom'
-require './nameable'
-require './book'
-require './rental'
+#!/usr/bin/env ruby
```
Comment (mohametalmeari) [inline main.rb:1]: I added this line but still not working

File: main.rb [hunk 2]
```diff
@@ -1,67 +1,46 @@
...
+require './app'
+MyApp = App.new
+def show_list
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
```
Comment (DuaneDave) [inline main.rb:10]: Hey @mohametalmeari, good job so far, every operation from the console works except that when I try to create a rental, it throws the error seen in the below screenshot. I recommend addressing this issue in order to make the app work fully.

image

Please Note: the error exists irrespective of the format of the date provided

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @mohametalmeari 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ The main.rb file is executable
✔️The options in the console app is well arranged

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @mohametalmeari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohametalmeari** [comment]
Thank you for the review

**Pull request merged**